scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Api.scala

```diff
@@ -1,6 +1,64 @@
 package scala.meta
 package semantic
 
-private[meta] trait Api extends v1.Api
+import scala.meta.inputs._
 
-private[meta] trait Aliases extends v1.Aliases
+private[meta] trait Api extends Flags {
+  implicit class XtensionPositionAnchor(pos: Position) {
+    def toAnchor: Anchor = pos.input match {
+      case scala.meta.inputs.Input.File(path, _) =>
+        Anchor(path, pos.start.offset, pos.end.offset)
+      case other =>
+        sys.error(s"unsupported input " + other)
+    }
+  }
+
+  implicit class XtensionSemanticEquality(tree1: Tree)(implicit m: Mirror) {
+    def ===(tree2: Tree): Boolean = scala.meta.internal.semantic.equality.equals(tree1, tree2)
```

Comment:
I think the difference between semantic and syntactic equality is abstract enough for most people. I'm not convinced symbolic operators like `<:<` help. The symbolic operators should be opt-in in my opinion through some `scala.meta.syntax._` import. One  benefit of alphanumerical names is that they are easier to search for on google and discuss about on gitter/SO.